servers/Azure.Mcp.Server/src/Program.cs

```diff
@@ -58,6 +63,29 @@ private static async Task<int> Main(string[] args)
             return 1;
         }
     }
+
+    private static void DisplayServerVersion()
+    {
+        var assembly = Assembly.GetEntryAssembly();
+        string version = "unknown";
+
+        if (assembly != null)
+        {
+            try
+            {
+                version = OpenTelemetryExtensions.GetServerVersion(assembly);
```

Comment:
In what cases would this not return not a value? 

If it's a case we expect, I'd be OK with catching and setting a default.. but if it's unexpected, maybe there's something wrong in the program?